ee/vellum_ee/workflows/tests/test_server.py

```diff
@@ -4,11 +4,18 @@
 from typing import Type, cast
 
 from vellum.client.core.pydantic_utilities import UniversalBaseModel
+from vellum.client.types.code_executor_response import CodeExecutorResponse
+from vellum.client.types.number_vellum_value import NumberVellumValue
 from vellum.workflows import BaseWorkflow
 from vellum.workflows.nodes import BaseNode
 from vellum_ee.workflows.server.virtual_file_loader import VirtualFileFinder
 
 
+@pytest.fixture
+def mock_open(mocker):
+    return mocker.patch("vellum.workflows.nodes.displayable.code_execution_node.utils.open")
```

Comment:
@m-abboud this is where we hand it off to you to solve the second blocker w.r.t `open`